pkg/envinfo/envinfo.go

```diff
@@ -280,8 +291,8 @@ func (esi *EnvSpecificInfo) IsSet(parameter string) bool {
 	return util.IsSet(esi.componentSettings, parameter)
 }
 
-// EnvInfoFileExists if the envinfo file exists or not
-func (esi *EnvSpecificInfo) EnvInfoFileExists() bool {
+// Exists returns whether the envinfo file exists or not
+func (esi *EnvSpecificInfo) Exists() bool {
```

Comment:
Shouldn't `EnvInfoFileExists` makes more sense than `Exists`

Reply:
No because it's part of the `LocalConfigProvider` interface which is also implemented by `LocalConfigInfo`.

Reply:
This is correct. I don't like the long names that are duplicating package or struct names that we have in some places.